plugins/entities/plugin.js

```diff
@@ -143,7 +148,9 @@
 				if ( config.entities && config.entities_processNumerical )
 					entitiesRegex = '[^ -~]|' + entitiesRegex;
 
-				entitiesRegex = new RegExp( entitiesRegex, 'g' );
+
+				// IE does not support unicode option in the Regex constructor (#4941).
+				entitiesRegex = CKEDITOR.env.ie ? new RegExp( entitiesRegex, 'g' ) : new RegExp( entitiesRegex, 'gu' );
```

Comment:
Lets make this code less repeatable:
```suggestion
				entitiesRegex = new RegExp( entitiesRegex, CKEDITOR.env.ie ? 'g' : 'gu' );
```